CHANGELOG.md

```diff
@@ -299,7 +305,8 @@ are in a single HTTP session, handle pagination and respects conformance
 
 Initial release.
 
-[Unreleased]: <https://github.com/stac-utils/pystac-client/compare/v0.7.0...main>
+[Unreleased]: <https://github.com/stac-utils/pystac-client/compare/v0.7.1...main>
+[v0.7.1]: <https://github.com/stac-utils/pystac-client/compare/v0.7.0...v0.7.1>
```

Comment:
I'm a little hung up on the two-dot versus three-dot comparison. Earlier comparisons use two dots.

Reply:
Three-dot is the [GitHub default](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons). Three-dot versus two-dot when working with tags does not appear to change the diff shown on GitHub. But using three-dot shows the commits, which is useful. I'd say change them all to be three-dot.